examples/next/pages/ui/components/ai/ai-conversation/amplify_outputs.json

```diff
@@ -0,0 +1,3 @@
+{
+    
```

Comment:
Might just be me, but seems weird to have an empty object on multiple lines, can is just be `{}`?

Reply:
will update